src/main/java/me/botsko/prism/actions/BlockAction.java

```diff
@@ -350,50 +350,52 @@ ChangeResult placeBlock(Player player, QueryParameters parameters, boolean isPre
         state.update(true);
         BlockState newState = block.getState();
         BlockActionData blockActionData = getActionData();
+        if(blockActionData != null ) {
```

Comment:
Codacy found an issue: [WhitespaceAround: 'if' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)